drivers/libusb.c

```diff
@@ -582,5 +570,9 @@ usb_communication_subdriver_t usb_subdriver = {
 	libusb_get_report,
 	libusb_set_report,
 	libusb_get_string,
-	libusb_get_interrupt
+	libusb_get_interrupt,
+        0,
+        0,
+        1,
+        1
```

Comment:
Cosmetic: indentation

Also given the "human-readable" names of values above, it seems useful to define meaningfully named macros for default values, to explain these zeroes and ones here and in other subdrivers that would use them.

Reply:
Not a strong preference, but I think just having a comment to the right of the 1's and 0's to indicate e.g. `hid_ep_in` would be clearer than a macro/enum. TBH we should be parsing the EP numbers from a descriptor anyway.

Reply:
@clepple @jimklimov thank you for review, fixed in the latest commit, added macros and comments